tetra/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:2.7
 
 WORKDIR /tetra/
 ADD setup.py ./
-ADD tetra tetra/
+ADD . tetra/
```

Comment:
Is it not common to have `setup.py` in the root dir? :thinking: 

Why move it?

Reply:
It's in the root dir of the api (the tetra folder) and there it's supposed to be one below the actual package (according to [this](https://setuptools.pypa.io/en/latest/userguide/quickstart.html))... 🤷‍♂️ - putting it in the root of the entire project seemed to be misleading IMO, since it has no effects on the UI or the project setup in general

Reply:
Alright, sense makin' :ok_hand: